File: webshop/src/main/java/com/webshop/model/dto/customer_order_article/CustomerOrderArticleViewDto.java
package com.webshop.model.dto.customer_order_article;

import com.webshop.model.Article;
import com.webshop.model.CustomerOrder;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class CustomerOrderArticleViewDto {

	private String orderArticleId;
	private int amountOfArticle;
	
	private CustomerOrder orderId;
	
	private Article articleId;
}

File: frontend/officechair/src/app/models/customerOrderArticleDto/customerOrderArticleViewDto.ts
import { ArticleViewDto } from "../articleDto/articleViewDto";
import { CustomerOrderViewDto } from "../customerOrderDto/customerOrderViewDto";

export class CustomerOrderArticleViewDto {

    orderArticleId: String;
	amountOfArticle: number;
	
	customerOrderArticle: CustomerOrderViewDto;
	
	article: ArticleViewDto;

	constructor() {
        this.orderArticleId = "";
        this.amountOfArticle = 0;
        this.customerOrderArticle = new CustomerOrderViewDto;
        this.article = new ArticleViewDto;
        
    }

}